docs/agents/sematext-agent/kubernetes/helm.md

```diff
@@ -0,0 +1,108 @@
+## Introduction
+
+This chart installs the Sematext Agent to all nodes in your cluster via a `DaemonSet` resource.
+
+## Prerequisites
+
+- Kubernetes 1.13+
```

Comment:
Given changes in k8s and SC-13489 is this version correct?  YG will know.

Reply:
Yes, the work there does not affect the minimum version we support